src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoN.java

```diff
@@ -16,6 +16,10 @@
 /**
  * HDF5 File backed Panel of Normals data structure.
  *
+ * Several attributes are stored transposed, in order to dodge a very slow write time in HDF5.
```

Comment:
Don't just say transposed -- say that samples are rows and targets are columns or whatever.